episodes/backward-design.md

```diff
@@ -56,7 +56,14 @@ By the end of this session, participants should be able to...
 
 ## Homework/Reading for Next Week
 
-- FIXME
+- [CLDT episode: Identifying Your Target Audience](https://carpentries.github.io/lesson-development-training/03-audience.html)
+- [CLDT episode: Defining Lesson Objectives/Outcomes] https://carpentries.github.io/lesson-development-training/05-objectives.html
+- Section 3 (_Formal curriculum design_) of [_Course design: Considerations for trainers - a Professional Guide_](https://doi.org/10.7490/f1000research.1118395.1) by Allegra Via et al, 2020[^1]
+- [Kirschner et al. 2006](https://carpentries.github.io/lesson-development-training/reference.html#litref)
+- [Lujan & DiCarlo 2005](https://carpentries.github.io/lesson-development-training/reference.html#litref)
+- (**Optional**) Chapter 1, How Does Students’ Prior Knowledge Affect Their Learning?, of Ambrose et al. 2010.↩︎
+- (**Optional**) Chapter 2, How Does the Way Students Organize Knowledge Affect Their Learning?, of Ambrose et al. 2010.↩︎
+
```

Comment:
I copied this list from the reading list of the new episode to be inserted after this one but may have gotten things wrong 😊 @tobyhodges as I did two commits on this (the second one after I realised the new episode is still a PR).

Reply:
Ah ok, got it. I think it's best to keep it as a FIXME until @mkuzak has finalised his draft in #19, just in case the reading list changes there before the PR is merged.

Reply:
Yes, best to wait I agree.